spec/features/system/admin/instance/user_management_spec.rb

```diff
@@ -87,7 +87,7 @@ def search_for_users(query, click: true)
       end
 
       scenario 'I can search users by email' do
-        random_instance_user = InstanceUser.order('RANDOM()').first
+        random_instance_user = InstanceUser.human_users.order('RANDOM()').first
         search_for_users(random_instance_user.user.email)
 
         expect(page).to have_text(random_instance_user.user.name)
```

Comment:
> Can also update the `admin/instance/user_management_spec.rb` to check `.email` using `have_selector` instead of checking `.name` using `have_text` then.

Reply:
sorry apparently the change was just reflected on my local, and for some reason it failed to push here. I have pushed the proper change, now the test is to check email using `have_selector`